tester/Makefile

```diff
@@ -33,7 +33,7 @@ CFLAGS := -Werror -Wundef \
 	-DST_TESTER \
 	-DSEQUENCE_TRANSFORM_DEBUG=1 \
 	-DSEQUENCE_TRANSFORM_ENHANCED_BACKSPACE=1 \
-	-DSEQUENCE_TRANSFORM_RULE_SEARCH=1 \
+	-DSEQUENCE_TRANSFORM_RULE_SEARCH=0 \
```

Comment:
RS_old still needs to be updated to work on the new trie. Until that is done, it must be disabled, or it will go off the rails.